manifests/init.pp

```diff
@@ -267,9 +267,10 @@
       require      => Class['certs::apache'],
     }
 
-    include foreman_proxy_content::pub_dir
   }
 
+  include foreman_proxy_content::pub_dir
```

Comment:
This is now deployed at all times with FPC right? If I read this correctly. That follows our assumption that if you are deploying an FPC you intend to have a pub dir.

Reply:
Yes, this is now "you're deploying f-p-c? you got a pubdir"

the only thing I'm curious about is whether `pub_dir.pp` actually needs that last else branch, aka: is fpc ever deployed without *any* pulp?

Reply:
In c363e7c0b068e7d2d06c5c0698c98f545c1c1f31 it was extracted, but it looks like 3d542ff831ebb167267de0e62eaa2a4fb63646f3 already introduced it. Back then it appeared to have been to expose the Pulp HTTP parts but that was superseded. So I think there isn't really a case where it makes sense.

Reply:
From a logic grouping perspective this is very closely related to our [bootstrap_rpm](https://github.com/theforeman/puppet-foreman_proxy_content/blob/master/manifests/bootstrap_rpm.pp). I wonder if that should actually include the pub dir code. That also feels unnatural.

A more logical thing might be to include bootstrap_rpm from pub_dir or even merge the two classes.

I'd be ok with doing that in a follow up PR.